contracts/src/StakeTable.sol

```diff
@@ -146,6 +146,9 @@ contract StakeTable is Initializable, InitializedAt, OwnableUpgradeable, UUPSUpg
     /// A zero amount would lead to a no-op.
     error ZeroAmount();
 
+    /// The caller is not the timelock
+    error NotTimelock();
```

Comment:
This looks like it's unused now?

Reply:
ah yes thanks https://github.com/EspressoSystems/espresso-network/commit/ca73a8188320ee73979ab5d349ebf1438097f230